public_html/wp-content/themes/campsite-2017/template-parts/content.php

```diff
@@ -15,6 +15,12 @@
 	<header class="entry-header">
 		<?php
 
+		if ( ( ! is_single() || is_singular( 'post' ) ) && has_post_thumbnail() ) : ?>
```

Comment:
```suggestion
		if ( ( ! is_single() || is_singular() ) && has_post_thumbnail() ) : ?>
```

Reply:
Assuming we don't need a separate feature flag here, couldn't this be simplified further to

```suggestion
		if ( has_post_thumbnail() ) : ?>
```

Reply:
@vedanshujain I think this is the only thing left to figure out what then I'd say it's ready to commit.

Reply:
I think this comment answers https://github.com/WordPress/wordcamp.org/pull/111#discussion_r281394415 . Going go ahead and implement discussed changes and deploy .

Reply:
Addressed in https://github.com/WordPress/wordcamp.org/pull/111#discussion_r281394415. Resolving